src/Params.mli

```diff
@@ -0,0 +1,25 @@
+(**********************************************************************
+ *                                                                    *
+ *                            OCamlFormat                             *
+ *                                                                    *
+ *  Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.  *
+ *                                                                    *
+ *  This source code is licensed under the MIT license found in the   *
+ *  LICENSE file in the root directory of this source tree.           *
+ *                                                                    *
+ **********************************************************************)
+
+module Cases : sig
```

Comment:
I'm still unclear what the reasoning is for this module. Wouldn't it be simpler to have `type cases` with `val get_cases`?